apps/molgenis-components/src/components/forms/BaseInput.vue

```diff
@@ -2,24 +2,44 @@
 export default {
   name: 'BaseInput',
   props: {
+    /**
+     * Unique identifier https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id
+     * Has no direct emx2 implementenation but can be constructend by concatinating the table name with the field name
+     */
     id: {
       type: String,
       required: true
     },
+    /**
+     * Name of the element. For example used by the server to identify the fields in form submits.
+     * relates to emx 2 columnName
+     */
```

Comment:
The name is more restrictive and used for communicating with the backend, the label can fallback to the name but should be focused on the human user. Not all the fields are generated form the schema, but all schema fields can be represented as a input ( except of heading :( )

Reply:
ok. EMX2 actually provides 'form elements' hence the 'heading'. But on topic: in most cases we now use v-model right? So I try to understand where we use the 'name'? Or is this a more theoretical consideration?

Reply:
v-model is used for binding the input value to the js(-model), the name is used for html form element. when using js to post js model data to the backend named form elements are not needed, would still suggest to keep it to have valid html forms. the name is also use by browsers for validation and tabbing details https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Reply:
yes, I get that (well, post that). But until now we never used 'name' so that is where my question came from :-). I don't mind this as long as we don't duplicate efforts. So defaulting this to the same as 'label' does the job